ppr-ui/src/App.vue

```diff
@@ -233,14 +233,22 @@ export default defineComponent({
           }
         }
 
-        // When we are authenticated, allow time for session storage propagation from auth, then initialize application
-        // (since we won't get the event from Signin component)
-        if (isAuthenticated.value) {
-          setTimeout(() => { onProfileReady(true) }, 1000)
-        }
+        authVerificationHandler()
       }
     })
 
+    /**
+     * When we are authenticated, allow time for session storage propagation from auth, then initialize application
+     * (since we won't get the event from signin component)
+     */
+    const authVerificationHandler = () => {
+      setTimeout(() => {
+        isAuthenticated.value && !!sessionStorage.getItem(SessionStorageKeys.CurrentAccount)
+          ? onProfileReady(true)
+          : authVerificationHandler()
+      }, 2000)
```

Comment:
That's the lowest we can go, is it noticeable delay? Or you wanted to have some buffer for the future?

Reply:
It's not really noticeable no. 
This function is in itself a buffer, in the rare event CURRENT ACCOUNT is missing, it will call itself again after 2seconds. 
I think if we were to add more time here, it would just delay that initial initialization when everything was good, which will be the scenario 9/10 times pretty sure.